### PR TITLE
[abseil] Upgrade to 20210324.2

### PR DESCRIPTION
**Pull request by srgbtl**

### CODE REVIEW

File: ports/abseil/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "abseil",
   "version-date": "2021-03-24",
-  "port-version": 1,
+  "port-version": 2,
```
Comment (JonLiu1993) [inline ports/abseil/vcpkg.json:4]:
```suggestion
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: ports/abseil/vcpkg.json (hunk 1 above)
Comment (srgbtl) [reply ports/abseil/vcpkg.json]: Not sure whether the port-version should be removed. According to the documentation, it should be present. Or I missed something?

Comment (srgbtl) [reply ports/abseil/vcpkg.json]: Or "version-date" should be changed to "2021-03-24.2"

**Review by JonLiu1993 — COMMENTED** [review]
Please run command "./vcpkg x-add-version abseil --overwrite-version" then commit the changes again

**Comment by JonLiu1993** [comment]
> Please run command "./vcpkg x-add-version abseil --overwrite-version" then commit the changes again

Ignore this suggestion because you have already done it

File: ports/abseil/vcpkg.json (hunk 1 above)
Comment (JonLiu1993) [reply ports/abseil/vcpkg.json]: > Not sure whether the port-version should be removed. According to the documentation, it should be present. Or I missed something?

When we update port,port-version should delete, or set to 0,you can refer to [this ](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/manifest-files.md#port-version)

Comment (JonLiu1993) [reply ports/abseil/vcpkg.json]: > Or "version-date" should be changed to "2021-03-24.2"

"version-date" do not need to change

File: ports/abseil/vcpkg.json [hunk 2]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "abseil",
-  "version-date": "2021-03-24",
-  "port-version": 1,
+  "version-date": "2021-03-24.2",
```
Comment (JonLiu1993) [inline ports/abseil/vcpkg.json:3]:
```suggestion
  "version-date": "2021-03-24",
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Comment by JonLiu1993** [comment]
@srgbtl ,Where is the _versions/a-/abseil.json_ file?

**Comment by srgbtl** [comment]
> @srgbtl ,Where is the _versions/a-/abseil.json_ file?

See: https://github.com/microsoft/vcpkg/pull/19357#pullrequestreview-722127313
Bot requested to remove changes there.

File: versions/a-/abseil.json [hunk 3]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
```
Comment (JonLiu1993) [inline versions/a-/abseil.json:3]: For another import point:
We expect that each time you update a port, you also update its version files.you can refer [this ](https://github.com/microsoft/vcpkg/blob/68a74950d0400f5a803026d0860f49853984bf11/docs/maintainers/maintainer-guide.md#update-the-version-files-in-versions-of-any-modified-ports)

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: ports/abseil/vcpkg.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (srgbtl) [reply ports/abseil/vcpkg.json:4]: @JonLiu1993 [this](https://github.com/microsoft/vcpkg/blob/5a271a9290282e09149401486f88dc106dc65b71/docs/specifications/versioning.md#port-version) doc says this:
```

[the remark this answers is not in the document]
Comment (srgbtl) [reply ports/abseil/vcpkg.json:4]: One more example: #19366

[the remark this answers is not in the document]
Comment (srgbtl) [reply ports/abseil/vcpkg.json:4]: CC: @NancyLi1013 @PhoebeHui

[the remark this answers is not in the document]
Comment (JonLiu1993) [reply ports/abseil/vcpkg.json:4]: Please delete  "port-version"

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Comment by srgbtl** [comment]
@JonLiu1993 removed. What should be done next? How to deal with errors reported in checks?

**Review by JonLiu1993 — APPROVED** [review]

File: ports/abseil/vcpkg.json [hunk 4]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "abseil",
-  "version-date": "2021-03-24",
-  "port-version": 1,
+  "version-string": "20210324",
```
Comment (PhoebeHui) [inline ports/abseil/vcpkg.json:3]:
```suggestion
  "version-date": "2021-03-24",
  "port-version": 2,
```

Comment (PhoebeHui) [inline ports/abseil/vcpkg.json:3]:
```suggestion
  "version-string": "20210324.2",
```

**Pull request merged**